docs(changelog): update changelog for tag v0.5.11 (#356)
tier4/caret_doc: docs/changelog.md

## CARET

### v0.5.11 <small>\_ Sep 30, 2024</small> {id = "0.5.11"}

> - **Update**: Fixed a problem where only one legend is displayed even if the maximum number of legends is exceeded. ([caret_analyze #523](https://github.com/tier4/caret_analyze/pull/523))

Reviewer: The content seems a bit inaccurate. How about the following text?

```suggestion
- **Update**: Fixed a probelm where a warning was incorrectly displayed when the maximum number of legends was not exceeded. ([caret_analyze #523](https://github.com/tier4/caret_analyze/pull/523))
```

- **Update**: Fixed mypy1.11 error untyped-overrides. ([caret_analyze #527](https://github.com/tier4/caret_analyze/pull/527))

> - **Update**: Fixed stackedbar when time reversal occurs. ([caret_analyze #517](https://github.com/tier4/caret_analyze/pull/517))

Reviewer: Please use stacked bar graph or `StackedBar`.

Author: Fixed in https://github.com/tier4/caret_doc/pull/356/commits/ad0751c8e11b35fc512449ec0dfb7c6656a329eb.

- **Update**: Fixed type_check_decorator. ([caret_analyze #513](https://github.com/tier4/caret_analyze/pull/513))

- **Update**: Refactoring of AssignContextReader class. ([caret_analyze #532](https://github.com/tier4/caret_analyze/pull/532))

> - **Update**: refactoring for the case of merging with source timestamp. ([caret_analyze #526](https://github.com/tier4/caret_analyze/pull/526))

Reviewer: The changes are not clear. Can you be more specific?

Author: Fixed in https://github.com/tier4/caret_doc/pull/356/commits/ad0751c8e11b35fc512449ec0dfb7c6656a329eb.
